config/permissions.tmpl.yml

```diff
@@ -35,6 +35,7 @@
 - ems-type:gce_network
 - ems-type:hawkular
 - ems-type:hawkular_datawarehouse
+- ems-type:monitoring
```

Comment:
@moolitayer looking at the other types they're technology-specific (ansible, azure, cinder, ec2, s3, foreman, gce, etc.).

I don't see anything generic (e.g. network, storage, cloud, infra).

Then why "monitoring" instead of probably "prometheus"?